packages/dx-react-scheduler-material-ui/src/templates/appointment-form/date-editor.jsx

```diff
@@ -16,36 +17,43 @@ const DateEditorBase = ({
   label,
   className,
   readOnly,
+  onValueChange,
   ...restProps
 }) => (
   <TextField
     label={label}
     className={classNames(classes.editor, className)}
-    value={value}
+    value={moment(value).format(moment.HTML5_FMT.DATETIME_LOCAL)}
     margin="normal"
     variant="filled"
     type="datetime-local"
     disabled={readOnly}
     InputLabelProps={{
       shrink: true,
     }}
+    onChange={({ target }) => onValueChange(target.value)}
     {...restProps}
   />
 );
 
 DateEditorBase.propTypes = {
   classes: PropTypes.object.isRequired,
-  value: PropTypes.string,
+  value: PropTypes.oneOfType([
```

Comment:
Just a thought. Should we allow using numbers here? It's useful for the [Unix time](https://en.wikipedia.org/wiki/Unix_time) format.